### PR TITLE
HW1-2-3-4-5-7-8-9: Serhei Poluyan

### PR DESCRIPTION
**Pull request by pevlens**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8).
- [ ] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [ ] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

**Comment by pevlens** [comment]
HW

File: src/task_1_1.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+a = 4.6
```
Comment (tamkovich) [inline src/task_1_1.py:1]: добавь описание задач в шапку

File: src/task_1_2.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from math import fabs
```
Comment (tamkovich) [inline src/task_1_2.py:1]: Сделай задание без использования math

File: src/task_1_3.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+length = 4
+square = length**2
```
Comment (tamkovich) [inline src/task_1_3.py:2]: https://github.com/tamkovich/tms-python-spring/blob/hw_reviews/src/Common.md пункт 24

File: src/task_1_4.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+last = (1, 2, 5, 8, 1, 5, 2, 6, 3)
+arithmetical_mean = sum(last) / len(last)
+product_numbers_1 = 1
+for i in range(len(last)):
+    product_numbers_1 = 1 * last[i] * product_numbers_1
+Geometric_Mean = product_numbers_1 / len(last)
```
Comment (tamkovich) [inline src/task_1_4.py:6]: именуй переменные с маленькой буквы

```suggestion
geometric_mean = product_numbers_1 / len(last)
```

File: src/task_1_5.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+from math import sqrt
```
Comment (tamkovich) [inline src/task_1_5.py:1]: без math

**Review by tamkovich — CHANGES_REQUESTED** [review]
.

**Comment by tamkovich** [comment]
Разбей на разные ветки домашки

**Pull request closed without merging**